test/tests/TestInstaller.hx

```diff
@@ -105,6 +106,19 @@ class TestInstaller extends TestBase {
 		assertTrue(scope.isLibraryInstalled(ProjectName.ofString("hx.signal")));
 	}
 
+	public function testInstallUrlChanges() {
+		installer.installFromHxml("git-deps-url-old.hxml");
```

Comment:
For this test, we should run `installVcsLibrary` rather than `installFromHxml`, since we are only testing the git functionality